build-system/compile/sources.js

```diff
@@ -42,6 +42,7 @@ const COMMON_GLOBS = [
   'node_modules/intersection-observer/intersection-observer.install.js',
   'node_modules/promise-pjs/package.json',
   'node_modules/promise-pjs/promise.mjs',
+  'node_modules/rrule/dist/es5/rrule.min.js',
```

Comment:
OMG, you got this to work!

![giphy-downsized](https://user-images.githubusercontent.com/26553114/86171075-255a2600-baea-11ea-9c25-c2389d0f26e0.gif)